content/docs/rfcs/38/README.md

```diff
@@ -0,0 +1,480 @@
+---
+slug: 38
+title: 38/LOGOS-CONSENSUS
+name: Logos Glacier Consensus Protocol
+status: raw
+category: informative
+tags: logos/consensus,implementation/rust, implementation/python, implementation/common-lisp
+editor: Mark Evenson <mark.evenson@status.im>
+created: 01-JUL-2022
+revised: <2022-08-05 Fri 23:49Z>
+contributors:
+    - Álvaro Castro-Castilla 
+---
+
+# Abstract
+
+We propose to replace Nakomoto consensus mechanisms with ones which
+afford execution by secure leaderless, decentralization.  We sketch a
+two-layer model for the practical execution of such a distributed
+consensus, in which an underlying binary decision mechanism is
+utilized to used to vote on the construction of a distributed,
+directed, acyclic graph.  We present the Glacier algorithm which
+provides a Byzantine fault tolerant implementation of the base binary
+decision mechanism.  We outline a taxonomy of Byzantine adversaries
+that seek to thwart this computation's "correctness".
+
+# Consensus Model
+
+Given an underlying binary consensus mechanism, one may quickly vote
+on the distributed, directed acyclic ledger graph of transactions.  If
+the underlying binary consensus mechanism is Byzantine fault tolerant,
+then one essentially gets the computation of the trust of the graph of
+transactions "for free".  The finalization of shared confidence in the
+values in a given sequence along this graph is isomorphic to the trace
+of a shared, trusted state machine evolution.  Such a state machine
+may perform an arbitrary computation from the class of "smart
+contract" artifacts by implementing a
+suitable model of transaction.  
+
+# Glacier 
+
+The Glacier consensus algorithm computes a yes/no decision via a set
+of distributed computational nodes.  Glacier is a leaderless
+probabilistic binary consensus algorithm with fast finality that
+provides good reliability for network and Byzantine fault tolerance.
+
+## Algorithm 
+
+Given a set of $n$ distributed computational nodes, a gossip-based
+protocol is presumed to exist which which allows members to discover,
+join, and leave a possibly maximally connected graph.  Joining this
+graph allows each node to view a possibly incomplete node membership
+list of all other nodes.  This view may change as the protocol
+advances, as nodes join and leave.  
+
+
+### Pre-requisites
+
+A proposal is formulated to which consensus of truth or falsity is
+desired.  Each node that participates starts the protocol with an
+opinion on the proposal, represented in the sequel as **YES**, **NO**,
+and **UNDECIDED**.
+
+     opinon
+        <-- choose local truth computation of  { YES, NO, UNDECIDED}
+    
+The algorithm proceeds in rounds for each node.  The liveness of the
+algorithm is severely constrained in the absence of timeouts for a
+round to proceed.
+
+### Setup
```

Comment:
I guess to name this section "Parameters" would more accurate.

Reply:
Changed to "Setup Parameters".  Distinguished between constants and variables in their use in the rounds.